### ISSUE TITLE
Logs are polluting JSON output

### ISSUE THREAD

**Issue by DavidBertet** [mataku/danger-ktlint#13]
We updated `ktlint` to latest version (0.50.0) recently, and Danger-ktlint started failing with a parsing error
```
!] Invalid Dangerfile file: unexpected token at 07:59:55.010 [main] WARN com.pinterest.ktlint.cli.internal.KtlintCommandLine - Lint has found errors than can be autocorrected using ktlint --format
...
```

Turns out, `ktlint` logs output are on `stdout` and are enabled with `info` by default.
 
They might have added a new log recently, but logs should be set to `none` as `danger-ktlint` is expecting json output

**Comment by DavidBertet** [comment]
They did add that log in `0.50.0`
https://github.com/pinterest/ktlint/commit/9754f5fca464f6c9ff7b829abaf5369bcc73d43a#diff-c8e91b74bbdc9243cc7527fbb1f76501990f6e2d4573ca064474cace6b31de9bR337

Issue has been filed on ktlint as well https://github.com/pinterest/ktlint/issues/2117

**Comment by mataku** [comment]
Released as v0.0.8 including this fix. Thanks for your help!
https://rubygems.org/gems/danger-ktlint/versions/0.0.8
